src/utils/yup-shema.ts

```diff
@@ -0,0 +1,188 @@
+import * as Yup from "yup";
+import { countryList } from "./country-list";
+
+const yupSchema = Yup.object().shape({
+  isAgree: Yup.boolean().isTrue("You must confirm acceptance"),
+  image: Yup.mixed()
+    .test("File should be select", "Image file should be select", (obj) => {
```

Comment:
```suggestion
    .test("File should be select", "Image file should be select", (fileList: FileList) => {
```
obj is not a good name :)